util/packaging/aur/chapel-git/.SRCINFO

```diff
@@ -0,0 +1,18 @@
+pkgbase = chapel-git
+	pkgdesc = Programming language designed for productive parallel computing at scale
+	pkgver = 1.27.0.7648.gbcc4ac3109
```

Comment:
Unsure exactly what you mean by "copied" it from something else?
The output of the pkgver() function automatically overwrites the pkgver line in PKGBUILD (I think the `makepkg` command probably feeds it to `sed` or something behind the scenes?).
As for what it means, I'm unsure on the central 4-digit part, but the very last part is `g<beginning of most recent commit hash`.

Reply:
When I made a hacky chapel-git for my own testing I just copied the version number from the other AUR project I was basing it on. Anyway since it will be overwritten in the PKGBUILD file by running `pkgver()`, it might be nice to have a comment to that effect next to the `pkgver=` line.